lib/std/os/windows/kernel32.zig

```diff
@@ -340,6 +340,8 @@ pub extern "kernel32" fn GetExitCodeProcess(
 // TODO: Already a wrapper for this, see `windows.GetCurrentProcess`.
 pub extern "kernel32" fn GetCurrentProcess() callconv(WINAPI) HANDLE;
 
+pub extern "kernel32" fn GetProcessId(Process: HANDLE) callconv(WINAPI) DWORD;
```

Comment:
Since this and `GetConsoleProcessList` are only used in the test code, and because of https://github.com/ziglang/zig/issues/1840, these bindings should be moved into `test/standalone/detached_child/main.zig`. The wrappers in `std.os.windows` can be deleted (and then just call the `kernel32` functions directly) or also moved into the test code.